src/gen/templates/task.t

```diff
@@ -0,0 +1,36 @@
+{% set file_name = name |  snake_case -%}
+{% set struct_name = file_name | pascal_case -%}
+to: "src/tasks/{{file_name}}.rs"
+skip_glob: "src/tasks/{{file_name}}.rs"
```

Comment:
because the file name is known in advance you can use just:
```
skip_exists: true
```
instead of `skip_glob`

Reply:
changed